cfpb/consumerfinance.gov
# New homepage soft launch

After lots of user testing and iterations, we're ready to roll out the homepage behind a feature flag. We'll remove the flag, clean up the templates and remove the hardcoded content in a subsequent release.

## Additions

- New homepage layout
- Hardcoded content
- Pics 'n stuff

## How to test this PR

1. `./frontend.sh`
1. Hard refresh http://localhost:8000/?home_page_2021=True&home_page_2021_minimal=True and you should see a new homepage layout.

## Screenshots


## Notes and todos

-


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
